plugins/digitalocean/requirements.txt

```diff
@@ -1 +1,3 @@
 resotolib==2.0.0a18
+boto3==1.20.32
+requests==2.27.1
```

Comment:
resotolib already defines this dependency and version. Since both of them can evolve, we should either use one location or do not define an exact version (but define as minimal version`>=`).